### PR TITLE
Update Sentry Python SDK to version 2.15.0

### PR DESCRIPTION
**Pull request by antonpirker**
And change all calls to `start_span(description="...")` to `start_span(name="...")` because `description` is deprecated and will be replaced with `name` in version 3.0.0.

### CODE REVIEW

File: src/sentry/api/endpoints/organization_events_trends.py [hunk 1]
```diff
@@ -261,9 +261,11 @@ def get_snql_function_aliases(trend_columns: TrendColumns, trend_type: str) -> d
                         if trend_type == IMPROVED
                         else aggregate_filter.operator
                     ),
-                    -1 * aggregate_filter.value.value
-                    if trend_type == IMPROVED
-                    else aggregate_filter.value.value,
+                    (
+                        -1 * aggregate_filter.value.value
+                        if trend_type == IMPROVED
+                        else aggregate_filter.value.value
```
Comment (szokeasaurusrex) [inline src/sentry/api/endpoints/organization_events_trends.py:267]: These changes (and the rest of the similar ones in this file) seem unrelated to `start_span`, maybe they got auto-formatted? Would consider splitting into separate PR, but not super important

**Review by szokeasaurusrex — APPROVED** [review]

**Review by sentrivana — APPROVED** [review]

**Comment by asottile-sentry** [comment]
fwiw this appears to be a dupe of https://github.com/getsentry/sentry/pull/78394

**Comment by antonpirker** [comment]
closing because of duplicate
